verification/spec/connection-handshake/ICS3Types.tla

```diff
@@ -0,0 +1,296 @@
+----------------------------- MODULE ICS3Types -----------------------------
+
+(***************************************************************************
+
+    This module is part of the TLA+ high-level specification for the
+    IBC Connection Handshake protocol (ICS3).
+    
+    This module includes common domain definitions that other modules will
+    extend.
+
+
+ ***************************************************************************)
+
+EXTENDS Naturals
+
+CONSTANTS MaxHeight,
+          AllConnectionIDs,
+          AllClientIDs
+
+
+(******************************* InitClients *****************************
+     A set of records describing the possible initial values for the
+     clients on a chain.
+     
+     A client record contains the following fields:
+
+     - consensusStates -- a set of heights, each height being a Nat 
```

Comment:
I find it a bit confusing to use term Nat here as we are constraining set of heights with MaxHeight. I would suggest just not to refer to "type" of the field, just say what it represents.